spm_funcs.py

```diff
@@ -54,8 +54,17 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+
+
+    img = nib.load(fname)
+    data = img.get_fdata()
+    spm_vals = []
+
+    for i in range(data.shape[-1]):
+        vol_mean = spm_global(data[..., i])
+        spm_vals.append(vol_mean)
+
+    return spm_vals  # Return the result.
```

Comment:
What is the type of `spm_vals`?

If you read the docstring (line 54 above) it says `array`, which is very unclear but seems to refer to a `numpy.array`, however, you return a basic `list` here. Consider converting it to array before returning or else downstream methods could fail. For instance, with the current implementation you would not be able to do:
```Python
average_spm = get_spm_globals(nifti_file).mean()
```